plugins/modules/zos_copy.py

```diff
@@ -65,14 +65,14 @@
       - Specifies whether a backup of destination should be created before
         copying data.
       - When set to C(true), the module creates a backup file or data set.
-      - The backup file name will be return on either success or failure of
+      - The backup file name will be returned on either success or failure of
         module execution such that data can be retrieved.
     type: bool
     default: false
     required: false
   backup_file:
     description:
-      - Specify the USS file name or data set name for the destination backup.
+      - Specify an unique USS file name or data set name for the destination backup.
```

Comment:
same comment as rst, not a big deal if we leave it

Reply:
Fixed.